tap_freshdesk/schemas/ticket_fields.json

```diff
@@ -0,0 +1,79 @@
+{
+    "type": "object", 
+    "properties": {
+        "id": {
+            "type": ["null", "integer"]
+        }, 
+        "default": {
+            "type": ["null", "boolean"]
+        },
+        "description": {
+            "type": ["null", "string"]
+        },
+        "label": {
+            "type": ["null", "string"]
+        },
+        "name": {
+            "type": ["null", "string"]
+        },
+        "position": {
+            "type": ["null", "integer"]
+        },
+        "required_for_closure": {
+            "type": ["null", "boolean"]
+        }, 
+        "type": {
+            "type": ["null", "string"]
+        },
+        "required_for_agents": {
+            "type": ["null", "boolean"]
+        },
+        "required_for_customers": {
+            "type": ["null", "boolean"]
+        },
+        "label_for_customers": {
+            "type": ["null", "string"]
+        },
+        "customers_can_edit": {
+            "type": ["null", "boolean"]
+        },
+        "displayed_to_customers": {
+            "type": ["null", "boolean"]
+        },
+        "portal_cc": {
+            "type": ["null", "boolean"]
+        },
+        "portal_cc_to": {
+            "type": ["null", "string"]
+        },
+        "choices": {},
```

Comment:
Why the type is not mentioned for `choices`?

Reply:
As per the [Freshdesk API documentation](https://developers.freshdesk.com/api/#ticket-fields), `choices` is the list of values. But, in the actual response, we observed different types of values including a `list of strings`, `plain object(dict),` an `object of the list`, etc. That's why due to uncertainty in the value of this field, we kept `{}` as datatype and generally we keep `{}` in such cases.